Makefile

```diff
@@ -187,6 +180,29 @@ oracle-database:
 		-v $(PWD)/script/oracle:/opt/oracle/scripts/setup \
 		quay.io/3scale/oracle:12.2.0.1-ee
 
+oracle-db-setup: ## Creates databases in Oracle
+oracle-db-setup: oracle-database
+	MASTER_PASSWORD=p USER_PASSWORD=p ORACLE_SYSTEM_PASSWORD=threescalepass NLS_LANG='AMERICAN_AMERICA.UTF8' DATABASE_URL="oracle-enhanced://rails:railspass@127.0.0.1:1521/systempdb" bundle exec rake db:drop db:create db:setup
+
+## Postgres database containerization
+postgres-database: ## Starts Postgres database container
```

Comment:
I wonder, most of the OS have postgres shipped with a package manager.
So is this really needed?

Reply:
Just added if the person didn't want to install it  😄. I can remove it also and accept the DATABASE_URL as argument, so the user can pass it using any installation method used. Wdyt @hallelujah and @Martouta ?

Reply:
Good for me 👍 and we follow the same approach for all DBs 😄 but i am opened to opinions about this 😄